### ISSUE TITLE
Eliminate Vercel Analytics

### ISSUE THREAD

**Issue by ArcanaOfSouls** [systemic-chaos/currently-fronting#240]
This is actually already done, but documentation needs to be updated to reflect that. The fact that we [encouraged use of uBlock Origin on our own web app](https://github.com/systemic-chaos/currently-fronting#privacy-policy) says a lot about how we feel about that Vercel Analytics as a platform, tbh.

The only reason it wasn't disabled sooner is that even once cancelled, the subscription period must expired before data points cease being tracked; which is extra concerning (even from a finance perspective, since Vercel was billing us per 1000 data points&mdasha value that wasn't really a problem till the app took off, to our pleasant surprise). The subscription finally ran out, which means the data points have stopped bing collected. Good riddance.

If anyone with uBlock Origin enabled in their browser (like us) notices anything still blocked by that extension, *please* file an issue. We respect everyone's right to privacy completely and want to make sure we don't make a similar mistake in the future.

This is the uBlock Origin report for our web app at present (using our system's page as an example), indicating 0 items blocked:  
![nothing to see here](https://user-images.githubusercontent.com/1747037/211163702-993ec2e6-1da9-4da7-a820-26583f2bad23.png)

Versus my feeling when GitHub has 77 items blocked right now... Oof:
![image](https://user-images.githubusercontent.com/1747037/211163785-95e623cd-defc-4f85-8722-78e43c82a075.png)

Perhaps one day we'll migrate away from GitHub, but that seems unlikely to occur in the near future. The least we can do is live up to our own privacy advocacy principles on the web app deployed from this repository.
